### PR TITLE
add method toArray()

### PR DESCRIPTION
**Pull request by luk156**

### CODE REVIEW

File: src/backbone.treemodel.js [hunk 1]
```diff
@@ -28,7 +28,17 @@
 			if(children.length) jsonObj.nodes = children;
 			return jsonObj;
 		},
+		toArray: function(){
+			if (!this.isRoot())
```
Comment (EnotionZ) [inline src/backbone.treemodel.js:33]: Is there a reason for excluding the root node as a member of the array?

File: src/backbone.treemodel.js [hunk 2]
```diff
@@ -28,7 +28,17 @@
...
+				var arr = [this];
+			else
+				var arr = [];
+			$.each(this._nodes.models,function(i,n){
```
Comment (EnotionZ) [inline src/backbone.treemodel.js]: I know Backbone is moving away from jQuery as a dependency. `this._nodes` is a Backbone Collection, `this._nodes.each` is preferable here.

**Comment by EnotionZ** [comment]
can you write a test for this in `test/backbone.tree_test.js`

**Comment by luk156** [comment]
sorry but i'm a junior developer, could you help me writing test?

**Comment by EnotionZ** [comment]
closing this as it's now implemented in c3c794f11011ce27ee924595ed85d9f89a6a2b7e

**Pull request closed without merging**